localization/xliff/enu/constants/localizedConstants.enu.xlf

```diff
@@ -551,6 +551,15 @@
         <trans-unit id="doNotInstall">
             <source xml:lang="en">Do not install</source>
         </trans-unit>
+        <trans-unit id="input">
```

Comment:
Shouldn't these be updated in the package.nls.json after re-running build?

Reply:
No, these live in the LocalizedConstants file. package.nls.json only has extension commands with `mssql.`. Thats why I update the location where they come using from `import * as LocalizedConstants from '../constants/localizedConstants';` 

I also did re-run build and lint this is what I pushed.

Reply:
Pretty sure that isn't how it works for the MSSQL extension - vscode-nls doesn't understand xlf files as far as I know. It needs the strings to be in an nls.json file to be picked up. The task that @smartguest added is supposed to be doing that (reading in xlf and then writing the updated package.nls.json) I thought.

@smartguest am I missing something here?

Reply:
These are localized strings, not commands. I discussed this with Alex 
the localizedPackage.json.enu.xlf updates package.nls.json 
and
the localizedConstants.enu.xlf updates the localizedConstants.ts 

This is how it is set up.

Reply:
Ok, I have no idea how the localization is supposed to work for stuff inside the extension. We generate the package.nls.json from the localizedPackage.json.enu.xlf but don't seem to do anything for any other xlf files (such as localizedConstants.enu.xlf)

So I guess this is expected but I'm curious how these strings are even being localized for the actual release then. It seems like that's just broken at this time.

Let's see what @smartguest has to say about this...

Reply:
The localizedConstants.enu.xlf file is to update the localizedConstants.ts file essentially yes @Charles-Gagnon they have a localize call which is processed by the compile-src gulp task, (this is how it was before I made any changes to the localization, my change just made it so that package.nls.json was generated in a similar way).

Reply:
Ok, thanks. I'll follow up separately since this thing seems to be broken for these strings currently.